sites/fast-component-explorer/build/convert-readme.js

```diff
@@ -33,6 +36,18 @@ const md = new MarkdownIt({
     xhtmlOut: true,
 });
 
+/**
+ * Function to determine the correct href path if
+ * it uses a relative path
+ */
+function normalizeLink() {
+    return `href="${webComponentWorkspacePath}`;
+}
+
+function normalizeMarkdown(markdown) {
+    return markdown.replace(/href="\.\.\/\.\.\/\.\.\//gm, normalizeLink);
+}
```

Comment:
It appears that the trailing `"` is missing in both of these.

Reply:
That's on purpose, I'm not capturing the full href just the first relative part

Reply:
party, good to go 👍